src/js/components/RadioButton/RadioButton.js

```diff
@@ -71,6 +73,12 @@ const RadioButton = forwardRef(
         }}
         focus={focus}
         focusIndicator={focusIndicator}
+        onFocus={() => {
+          setFocus(true);
+        }}
+        onBlur={() => {
+          setFocus(false);
+        }}
```

Comment:
To match the handlers below
```suggestion
        onFocus={() => setFocus(true)}
        onBlur={() => setFocus(false)}
```